src/main/webapp/script.js

```diff
@@ -359,33 +359,75 @@ async function getRepList() {
     for (var i = 0; i < offices.length; i++) {
         for (number of offices[i]["officialIndices"]) {
             var bool = await checkIfRepInDatastore(officials[number]["name"]);
```

Comment:
i guess I missed this in an earlier review but please use a better name than bool